src/main/java/com/github/webicitybrowser/webicity/renderer/backend/html/HTMLRendererBackend.java

```diff
@@ -52,8 +60,14 @@ private void parseAndTimeDocument(Connection connection) throws IOException {
 	}
 	
 	private void parseDocument(Reader inputReader) throws IOException {
+		FetchEngine fetchEngine = new FetchEngineImp(new HTTPFetchConnectionPool());
+		RendererContext rendererContext = new RendererContextImp(new ResourceAssetLoader(), fetchEngine);
+
+		TagActions tagActions = new ActionsRegistry();

```

Comment:
Might be good to make a method called createTagActions, as this will definitely grow in the future